core/pom.xml

```diff
@@ -39,7 +39,7 @@ THE SOFTWARE.
 
   <properties>
     <staplerFork>true</staplerFork>
-    <stapler.version>1.250</stapler.version>
+    <stapler.version>1.252</stapler.version>
```

Comment:
Now I wonder why I didn't ask about this when reviewing: was this related to Enforcer Upper Bound Deps? Shouldn't it have a dedicated PR? 

Anyway, it revealed https://github.com/stapler/stapler/pull/128 for us, an internal plugin that does not compile anymore.